Scenarios/Cortana/README.md

```diff
@@ -7,6 +7,17 @@ These are the basic steps to add voice command functionality and integrate Corta
 1. **Create a Voice Command Definition (VCD) file and register it when the app is launched.** The VCD file is an XML document that defines all the spoken commands that the user can say to initiate actions or invoke commands when activating your app. For more information, see [VCD elements and attributes v1.2](https://msdn.microsoft.com/library/windows/apps/dn706593).
 2. **Handle the activation-by-voice command.** You may want to handle launch differently depending on the method by which a user launched your app.
 
+## Running the Sample
+1. Open the CortanaSample-WinStore10.sln Visual Studio solution.
+2. Ensure that the project includes the CortanaSample\CortanaSampleCommands.xml VCD file. If it does not, add it by selecting **Project** > **Add Existing Item**.
+3. Build and deploy the application to your phone or PC.
+4. Start the sample app for the first time, which should automatically register it with Cortana. **Close it immediately.**
```

Comment:
As supposed to saying it should automatically register...  How about:

4. Start the sample app in order to register the VCD file with Cortana. **Close sample app.**